contracts/Lending.sol

```diff
@@ -278,4 +278,9 @@ contract Lending {
     {
         interestRate = _interestRate;
     }
+
+    function withdrawAll(address ownersAddress) public payable onlyOwners isValidAddress(ownersAddress){
```

Comment:
`ownersAddress` here is not used because you withdraw to msg.sender. so I think you can remove the params and also isValidAddress

Reply:
oh yeah sure i forgot to remove that